.vib/grafana-operator/goss/grafana-operator.yaml

```diff
@@ -1,6 +1,6 @@
 command:
   check-app-version:
```

Comment:
Instead of `check-app-version`I'd use other name, for example: `chek-binary`, because it is not checking the version.